src/Component/Symfony/Request/State/TwigResponder.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Symfony\Request\State;
+
+use Sylius\Component\Resource\Context\Context;
+use Sylius\Component\Resource\Context\Option\RequestOption;
+use Sylius\Component\Resource\Metadata\CollectionOperationInterface;
+use Sylius\Component\Resource\Metadata\CreateOperationInterface;
+use Sylius\Component\Resource\Metadata\DeleteOperationInterface;
+use Sylius\Component\Resource\Metadata\HttpOperation;
+use Sylius\Component\Resource\Metadata\Operation;
+use Sylius\Component\Resource\Metadata\UpdateOperationInterface;
+use Sylius\Component\Resource\State\ResponderInterface;
+use Sylius\Component\Resource\Symfony\Routing\RedirectHandler;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Twig\Environment;
+
+final class TwigResponder implements ResponderInterface
+{
+    public function __construct(
+        private RedirectHandler $redirectHandler,
+        private ?Environment $twig,
```

Comment:
Do we need to allow for null env?

Reply:
yes Twig, we have smoke tests without Twig to place it on dev requirements later.